sound/soc/intel/boards/sof_pcm512x.c

```diff
@@ -244,8 +249,9 @@ static struct snd_soc_dai_link *sof_card_dai_links_create(struct device *dev,
 	links[id].dpcm_playback = 1;
 	/*
 	 * capture only supported with specific versions of the Hifiberry DAC+
-	 * links[id].dpcm_capture = 1;
 	 */
+	if (sof_pcm512x_quirk & SOF_PCM512X_ENABLE_SSP_CAPTURE)
+		links[id].dpcm_capture = 1;
```

Comment:
this seems to indicate, that without this patch SSP capture on UP2 shouldn't be working. For a reason unknown to me it's working without this patch, and it's also working with it.

Reply:
I don't know how it's possible to capture from a link that is explicitly not configured for capture. There must be a side effect or something. Maybe we also downgraded the message to throw an error but kept going to deal with other platforms, I have a vague memory of some changes related to this dpcm_capture flag.